internal/pkg/conv/map.go

```diff
@@ -0,0 +1,56 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package conv
+
+import "fmt"
+
+// FixMap changes the map[interface{}]interface{} to map[string]interface{} recursively
+// This is needed to convert YAML deserializer result into a JSON compatible result
+func FixMap(m map[string]interface{}) map[string]interface{} {
```

Comment:
This feels like it could have a better name, like YAMLMapToJSONMap or something to imply when it should be used.

Also the input being `map[string]interface{}` doesn't match the comment which implies it should be `map[interface{}]interface{}`.